git-subsplit.sh

```diff
@@ -334,7 +334,6 @@ subsplit_update()
 
 	if [ -n "$VERBOSE" ];
 	then
-		echo "${DEBUG} git fetch -q origin"
```

Comment:
Haha oops, was coping this from a modified version, sorry.

And yes, it specifically says so in the docs: "-t, --tags: Fetch all tags from the remote [...] *in addition to* whatever else would otherwise be fetched."